src/js/components/Layer/StyledLayer.js

```diff
@@ -197,11 +197,22 @@ const KEYFRAMES = {
   },
 };
 
-const getAnimationStyle = (animate = true, position, full) => {
-  const defaultAnimation = css`animation: ${KEYFRAMES[position][full]} 0.2s ease-in-out forwards`;
-
-  return animate ? defaultAnimation : '';
-}
+const getAnimationStyle = (props, position, full) => {
+  let animation =
+    props.animation !== undefined ? props.animation : props.animate;
```

Comment:
The default value of `animation` is a valid string, so I think it will never land on the option of `props.animate` and will potentially create a backward compatibility issue if someone assigned `animate={false}`

Reply:
We don't actually set a default value for `animation` anywhere. So, behaviorally, setting only `animate={false}` should still work as before.